.gitignore

```diff
@@ -1,3 +1,4 @@
 __pycache__
 build
 build?
+fork-notes
```

Comment:
Can you remove this line?  I don't think it was meant to be included.

Reply:
Done